doc/installationguide.md

```diff
@@ -227,8 +226,8 @@ used for the same purpose. For instance:
     any unexpected error.
 -   **singleConfigurationMode**: enables the Single Configuration mode for backwards compatibility (see description in
     the Overview). Default to false.
--   **timestamp**: if this flag is activated:
-    -   For NGSIv1/NGSIv2, the IoT Agent will add a `TimeInstant` metadata attribute to all the attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
+-   **timestamp**: if this flag is activated, the IoT Agent will add a 'TimeInstant' metadata attribute to all the
+    attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
```

Comment:
Diff shows:

![imagen](https://user-images.githubusercontent.com/1534240/87909798-32bf5c00-ca69-11ea-95e5-0d69f6ef586d.png)

Is correct with the current wording? I think two bullets (one for NGSIv1/NGSIv2 and other for NGSI-LD) looks better.

Reply:
Fixed 21a657a